tools/docker/healthcheck/Dockerfile

```diff
@@ -1,16 +1,31 @@
 # Stage 1 - Build elixir release of ockam_healthcheck elixir app
-FROM ghcr.io/build-trust/ockam-builder@sha256:5ab42598e35509cad3ea9c1e1bd0ed135ed1340c6ae44b762b1c8bbec31d5c68 as elixir-app-release-build
+FROM hexpm/elixir:1.14.5-erlang-24.3.4.11-debian-bullseye-20230227-slim AS elixir-app-release-build
+
+# install build dependencies
+# FIXME: we should use precompile .so instead of requiring the rust compiler
+RUN apt-get update; \
+  apt-get install -y --no-install-recommends \
+  ca-certificates \
+  openssl \
+  libncurses5 \
+  git \
+  curl \
+  g++
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain 1.71.1 -y
+ENV PATH=/root/.cargo/bin:$PATH
+
 COPY . /work
 RUN set -xe; \
-    cd implementations/elixir; \
-    make build; \
-    cd ockam/ockam_healthcheck; \
-    MIX_ENV=prod mix release;
-
+    cd /work/implementations/elixir/ockam/ockam_healthcheck; \
+    MIX_ENV=prod OCKLY_PRECOMPILED_LIB=false mix local.hex --force; \
+    MIX_ENV=prod OCKLY_PRECOMPILED_LIB=false mix local.rebar --force; \
+    MIX_ENV=prod OCKLY_PRECOMPILED_LIB=false mix deps.get; \
+    MIX_ENV=prod OCKLY_PRECOMPILED_LIB=false mix release;
 
 # TODO: Use distroless container after https://github.com/elixir-lang/elixir/issues/11942 is closed
 # Stage 2 - Create container and copy executables in above step
-FROM debian:11.1-slim@sha256:312218c8dae688bae4e9d12926704fa9af6f7307a6edb4f66e479702a9af5a0c
+FROM debian:bullseye-20230227-slim AS app
```

Comment:
Let's pin this image version `debian:bullseye-20230227-slim@sha256:403e06393d6b9dcb506eeef2adba9e30a97139c54e4c90d55254049f7d224081`